### ISSUE TITLE
about loss and mIou

### ISSUE THREAD

**Issue by vincennnnt** [jfzhang95/pytorch-deeplab-xception#23]
Sorry to bother u，i just download the voc2012，and i run train.py，but the loss is very high，about 20000+，and the mIou is about 39%.Is that because epoch that i use is 200.（too little？）By the way，the number of training pictures is1400+，the val pictures is 1560.

**Comment by vincennnnt** [comment]
Okay,thank u for ur reply!

**Comment by jfzhang95** [comment]
You are welcome.
